s3-source-connector/src/integration-test/java/io/aiven/kafka/connect/s3/source/IntegrationTest.java

```diff
@@ -172,17 +179,21 @@ void bytesTest(final TestInfo testInfo) throws IOException {
         assertThat(testBucketAccessor.listObjects()).hasSize(5);
 
         // Poll messages from the Kafka topic and verify the consumed data
-        final List<String> records = IntegrationBase.consumeMessages(topicName, 4, connectRunner.getBootstrapServers());
+        final List<String> records = IntegrationBase.consumeByteMessages(topicName, 4,
+                connectRunner.getBootstrapServers());
 
         // Verify that the correct data is read from the S3 bucket and pushed to Kafka
-        assertThat(records).containsOnly(testData1, testData2);
+        assertThat(records).map(String::new).containsOnly(testData1, testData2);
```

Comment:
Is  `String::new` necessary?

Reply:
Ah, right, perhaps an artifact from rebase, thanks.